src/excmds.ts

```diff
@@ -1869,6 +1869,38 @@ export async function viewcontainers() {
             .replace(/#/g, "%23")
             .replace(/ /g, "%20")
 }
+
+/** Reopens current tab in specified container.
```

Comment:
Please mention that the tab is not duplicated at the history is lost. (That is what happens, right?).

Reply:
Will do, that is what happens.

Reply:
It might be worth navigating back in the tab before closing it, otherwise `u` will get you trapped in a loop.